### ISSUE TITLE
Benchmark for React is wrong, you run React in a dev mode

### ISSUE THREAD

**Issue by istarkov** [taobaofed/rx-ssr-compare#2]
Just add NODE_ENV=production to server build webpack config

```javascript
  plugins: [
    new webpack.DefinePlugin({
      'process.env': {
        NODE_ENV: '"production"'
      }
    })
  ]
```

And run test as 
`NODE_ENV=production node ./renderToString.js`

And you will get near same timings

```
rx: 56.581ms
react: 85.216ms
```

**Comment by yyx990803** [comment]
FYI this is addressed in #3 ;)

**Comment by istarkov** [comment]
Having that #3 is merged, can be closed
